lib/helpers/auditor.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = audit;
+
+/**
+ * Audits the req/res for the client. The audit will use bunyan's
+ * formatters. See bunyan-format for the user-friendly output.
+ *
+ * @param  {Object} err   The http error object.
+ * @param  {Object} req   The http request object.
+ * @param  {Object} res   The http response object.
+ *
+ * @returns {undefined} Does not return anything.
+ */
+function audit(err, req, res) {
+    req.headers = req._headers;
+    req.url = req.path;
+    var log = req.log;
+
+    var obj = {
+        remoteAddress: req.connection.remoteAddress,
+        remotePort: req.connection.remotePort,
+        req: req,
+        res: res,
+        err: err,
+        secure: req.secure
+    };
+
+    var level = _defaultLevelFn(res ? res.statusCode : 200);
+    var logFn = log[level] ? log[level] : log.info;
```

Comment:
My only comment on switching the log level based on the status code returned, is that it is no longer an auditor when it is WARN or ERROR. The server auditor only logs to INFO, it would be best for this to be consistent.